src/s1reader/s1_burst_slc.py

```diff
@@ -360,6 +389,68 @@ class represents a polynomial function of range
 
         return az_carrier_poly
 
+    def total_doppler(self, xstep=500, ystep=50):
```

Comment:
I am not a fun of `total_doppler`. Maybe we should be more explicit with the name of the function and call it `geometry_steering_doppler`. I know the doc string give an explanation but the name of the function sounds a bit confusing.

Reply:
ETAD ATBD uses the term "Doppler centroid frequency of focused data", so maybe `focused_doppler()`?

Reply:
Thanks Yunjun and Virginia. focused_doppler is technically correct and represent the total_doppler. However, I think ```geometry_steering_doppler``` is a bit more clear and tops specific. It may sound like one component of Doppler which is due to geometricaly steering, so I added ```and```, i.e., I went with ```geometry_and_steering_doppler``` if ok with you both.